qcodes_contrib_drivers/drivers/Tektronix/Keithley_6500.py

```diff
@@ -0,0 +1,56 @@
+from qcodes.instrument.visa import VisaInstrument
+from qcodes.instrument import InstrumentChannel
+from qcodes.utils.validators import Numbers
+from functools import partial
+
+
+class Keithley_Sense(InstrumentChannel):
+    def __init__(self, parent: VisaInstrument, name: str, channel: str) -> None:
+        valid_channels = ['VOLT', 'CURR', 'RES', 'FRES']
+        if channel.upper() not in valid_channels:
+            raise ValueError('Channel must be one of the following: {}'.format(', '.join(valid_channels)))
```

Comment:
Instead of using `.format`, we prefer f-string style. Could you please change them throughout the driver?